_config.yml

```diff
@@ -38,4 +38,4 @@ paginate: 3
 
```

Comment:
Thanks, @AAugustine! After some research and experimentation, here’s what I’ve determined:
- `site.name` is set in _config.yml to prevent an auto-generated title from displaying on the page. Here’s an example of what the README looks like if I change `name` to `title` as the warning suggests:

  <img width="1072" alt="Screen Shot 2021-05-28 at 4 21 51 PM" src="https://user-images.githubusercontent.com/17183625/120050060-e326b680-bfd0-11eb-870f-b7a88283e9ce.png">

  Here’s what it looks without `name` or `title` in the configuration file:

  <img width="1072" alt="Screen Shot 2021-05-28 at 4 17 40 PM" src="https://user-images.githubusercontent.com/17183625/120050001-b70b3580-bfd0-11eb-8330-dd54038996aa.png">

  And finally, here’s what it looks like with `name` left as-is:

  <img width="1072" alt="Screen Shot 2021-05-28 at 4 18 16 PM" src="https://user-images.githubusercontent.com/17183625/120050170-49133e00-bfd1-11eb-8c75-312016ce063c.png">

  This last one is the version we want, so I’ll leave this option as-is.
  
- Everything else in the configuration (including the pagination plugin and setting) appears to be unused and unneeded, so I went ahead and removed them.
- There’s another error we are going to get, which can be [safely ignored](https://github.com/github/pages-gem/issues/399#issuecomment-280186771): `GitHub Metadata: No GitHub API authentication could be found. Some fields may be missing or have incorrect data.`